examples/examples_nodejs_test.go

```diff
@@ -1080,6 +1080,33 @@ func TestRegress4568(t *testing.T) {
 	})
 }
 
+func TestRegress5219(t *testing.T) {
```

Comment:
Is there a way to make sure it is failing fast? It might eventually error due to a timeout.